components/config/src/lib.rs

```diff
@@ -97,6 +106,27 @@ impl Config {
         };
 
         config.build_timestamp = Some(Utc::now().timestamp());
+
+        // Convert the file glob strings into a compiled glob set matcher. We want to do this once,
+        // at program initialization, rather than for every page, for example. We arrange for the
+        // globset matcher to always exist (even though it has to be an inside an Option at the
+        // moment because of the TOML serializer); if the glob set is empty the `is_match` function
+        // of the globber always returns false.
+        let mut glob_set_builder = GlobSetBuilder::new();
+
+        if let Some(ref v) = config.ignored_content {
+            if v.len() > 0 {
```

Comment:
we can drop that `if` and get the same result no?

Reply:
I don't follow: I need to get a ref to the Vec within the Option. Or do you mean I should consume the ignored_content?

Reply:
you can skip the `len()` check

Reply:
Ah, of course. Done.